srcpkgs/fsearch/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'fsearch'
+pkgname=fsearch
+version=0.1beta2
+revision=1
+build_style=gnu-configure
+hostmakedepends="gettext gettext-devel glib-devel po4a automake autogen autoconf-archive pkg-config intltool glib-devel"
+makedepends="$(vopt_if gtk3 gtk+3-3 gtk+3-devel gtk+-devel)"
+depends="pcre hicolor-icon-theme"
```

Comment:
Does this really depend on `pcre`? (note this pkg in void just contains pcre{grep,test})

Reply:
Yes:
https://github.com/cboxdoerfer/fsearch#requirements
But i'm not sure, should i depend it on libpcre or pcre.

Reply:
Done!